build/yamls/antrea.yml

```diff
@@ -4038,7 +4038,7 @@ data:
     antreaProxy:
       # To disable AntreaProxy, set this to false.
       enable: true
-      # ProxyAll tells antrea-agent to proxy all Service traffic, including NodePort, LoadBalancer, and ClusterIP traffic,
```

Comment:
I feel no need to add "Option". Probably just "proxyAll tells"

Reply:
Done